Frontend/ui-common/src/Components/MobileNumberEditor/View.purs

```diff
@@ -0,0 +1,277 @@
+        else if config.countryCodeField.countryCodeOptionExpended then
+          [Anim.listExpandingAnimation $  listExpandingAnimationConfig config] 
+          else [Anim.listExpandingAnimation $  listClosingAnimationConfig config
+               ,Anim.fadeOut true])
+            $
+  scrollView[
+      height WRAP_CONTENT 
+    , width MATCH_PARENT
+    , margin config.countryCodeOptionConfig.margin
+  ][
+    linearLayout
+      [ height config.countryCodeOptionConfig.height
+      , width config.countryCodeOptionConfig.width
+      , background config.countryCodeOptionConfig.background
+      , orientation VERTICAL
+      , stroke config.countryCodeOptionConfig.stroke
+      , cornerRadius config.countryCodeOptionConfig.cornerRadius
+      ]
+      (mapWithIndex (\index item ->
+        linearLayout
+          [ height config.countryCodeOptionElementConfig.height
+          , width config.countryCodeOptionElementConfig.width
+          , onClick push $ const $ CountryCodeSelected item
+          , orientation VERTICAL
+          ]
+          [ linearLayout
+            [ height MATCH_PARENT
+            , width MATCH_PARENT 
+            ][textView $
+              [ text item.countryCode
+              , color config.countryCodeOptionElementConfig.leftTextColor
+              , margin config.countryCodeOptionElementConfig.leftTextMargin
+              ] <> FontStyle.paragraphText LanguageStyle
+              ,linearLayout
+                [ height WRAP_CONTENT
+                , weight 1.0
+                ][]
+             , textView $
+              [ text item.countryName
+              , color config.countryCodeOptionElementConfig.rightTextColor
+              , margin config.countryCodeOptionElementConfig.rightTextMargin
+              ] <> FontStyle.paragraphText LanguageStyle
+            ] 
+          , linearLayout
+            [ height config.countryCodeOptionElementConfig.lineHeight
+            , width MATCH_PARENT
+            , background config.countryCodeOptionElementConfig.lineColor
+            , margin config.countryCodeOptionElementConfig.lineMargin
+            , visibility if ((length getCountryCodesObj)-1) == index  && config.countryCodeOptionElementConfig.lineVisibility then GONE else VISIBLE 
+            ][]
+          ]
+        )(getCountryCodesObj)
+      )
+  ]
+
+listExpandingAnimationConfig :: Config -> AnimConfig
+listExpandingAnimationConfig config =  animConfig 
+          { fromScaleY = 0.0
+          , toScaleY = 1.0
+          , fromY = -100
+          , toY = 0
+          , repeatCount = (PrestoAnim.Repeat 0)
+          , ifAnim =  true
+          , duration = 200
+          } 
+
+listClosingAnimationConfig :: Config -> AnimConfig
+listClosingAnimationConfig config = animConfig
+          { fromScaleY = 1.0
+          , toScaleY = 0.0
+          , fromY = 0
+          , toY = -100
+          , repeatCount = (PrestoAnim.Repeat 0)
+          , ifAnim =  true
+          , duration = 200
+          }
```

Comment:
we can move this config to controller or if you are creating seperate module for config move it there